Marlin/ultralcd.cpp

```diff
@@ -32,6 +32,11 @@
 
 extern float zprobe_zoffset;
 
+#if ENABLED(AUTO_BED_LEVELING_UBL)
+  int UBL_has_control_of_LCD_Panel = 0;
+  volatile int G29_encoderDiff = 0; // This is volatile because it is getting changed at interrupt time.
+#endif
```

Comment:
Actually, I'm leaning towards more of these globals being incorporated into the `bed_leveling` (now renamed `unified_bed_leveling`) class… and for that class's members to all be made static — a singleton like Stepper and Planner.